android/build.gradle

```diff
@@ -51,11 +51,11 @@ android {
         disable 'GradleCompatible'
     }
     compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
+        sourceCompatibility JavaVersion.VERSION_17
+        targetCompatibility JavaVersion.VERSION_17
     }
     kotlinOptions {
-        jvmTarget = "1.8"
+        jvmTarget = "17"
     }
```

Comment:
We should probably use `jvmToolchain` instead

https://github.com/stripe-samples/terminal-apps-on-devices/blob/e235efbfd61eaef0be67a7385b1a3140d02beff9/app/build.gradle.kts#L29